openedx/features/course_rating/api/v1/serializers.py

```diff
@@ -0,0 +1,30 @@
+"""
+Serializers for course_rating
+"""
+from rest_framework import serializers
+
+from openedx.features.course_rating.models import CourseRating, CourseAverageRating
+
+
+class CourseRatingSerializer(serializers.ModelSerializer):
+    """
+    Serializer for "CourseRating" model
+    """
+    course_rating = serializers.SerializerMethodField()
+
+    class Meta:
+        model = CourseRating
+        fields = ["id", "course", "course_rating", "rating", "comment", "is_approved", "user", "moderated_by"]
```

Comment:
nit: be consistent with single quote in code and double quotes in docstring and comments.

Reply:
Done